services/galley/src/Galley/Cassandra/Access.hs

```diff
@@ -31,6 +31,7 @@ defAccess SelfConv (Just (Set [])) = [PrivateAccess]
 defAccess ConnectConv (Just (Set [])) = [PrivateAccess]
 defAccess One2OneConv (Just (Set [])) = [PrivateAccess]
 defAccess RegularConv (Just (Set [])) = defRegularConvAccess
+defAccess GlobalTeamConv _ = []
```

Comment:
```suggestion
defAccess GlobalTeamConv s = maybe [InviteAccess] Set.toList s
```
This seems to be in line with the last catch-all clause for a set with at least one element. If the input set is `Nothing`, team members should be able to join via an invitation.

Reply:
But you shouldn't be invited to the global team conv, right? It should be done automatically when you join a team?